docs/narr.rst

```diff
@@ -36,7 +36,7 @@ Usage within an Object field:
   >>> a.location = loc
   Traceback (most recent call last):
   ...
-  WrongContainedType: ([WrongType('foo', <type 'unicode'>, '__name__')], 'location')
+  SchemaNotCorrectlyImplemented: ([WrongType('foo', <type 'unicode'>, '__name__')], 'location')
```

Comment:
Thanks for the fix.

I'm OK with not adding a specific version pin for zope.schema for this upgrade since it "just" a doctest.